Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -1434,6 +1441,53 @@ private extension ChatChannelController {
             self?.state = error == nil ? .localDataFetched : .localDataFetchFailed(ClientError(with: error))
         }
     }
+
+    private func getLastReadMessageId(firstUnreadMessageId: MessageId) -> MessageId? {
```

Comment:
Because the backend only provides and requires lastReadMessageId, the controller is the one in charge of making the conversion both ways.
From the SDK point of view, getting the first unread message is the most relevant piece of information